tests/Loader/CucumberNDJsonAstLoaderTest.php

```diff
@@ -0,0 +1,186 @@
+                                'keyword' => 'out',
+                                'examples' => [
+                                    [
+                                        'location' => ['line' => 555],
+                                        'keyword' => 'exa',
+                                        'tableHeader' => [
+                                            'location' => ['line' => 666],
+                                            'cells' => [
+                                                ['value' => 'A'],
+                                                ['value' => 'B'],
+                                            ],
+                                        ],
+                                        'tableBody' => [
+                                            [
+                                                'location' => ['line' => 777],
+                                                'cells' => [
+                                                    ['value' => 'A1'],
+                                                    ['value' => 'B1'],
+                                                ],
+                                            ],
+                                            [
+                                                'location' => ['line' => 888],
+                                                'cells' => [
+                                                    ['value' => 'A2'],
+                                                    ['value' => 'B2'],
+                                                ],
+                                            ],
+                                        ],
+                                    ],
+                                ],
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+        ]);
+        $features = $this->loader->load($file);
+
+        $this->assertEquals(
+            [
+                new FeatureNode(
+                    'Feature with a valid Scenario',
+                    null,
+                    [],
+                    new BackgroundNode('Empty Background', [], 'bac', 222),
+                    [
+                        new ScenarioNode('Empty Scenario', [], [], 'sce', 333),
+                        new OutlineNode('Examples Scenario', [], [], [
+                            new ExampleTableNode([
+                                666 => ['A', 'B'],
+                                777 => ['A1', 'B1'],
+                                888 => ['A2', 'B2'],
+                            ], 'exa'),
+                        ], 'out', 444),
+                    ],
+                    'fea',
+                    'en',
+                    $file,
+                    111,
+                ),
+            ],
+            $features,
+        );
+    }
+
+    public function testOutlineTableBodyIgnoredWhenTableHeaderIsMissing(): void
```

Comment:
Think this test method name is out of date?

Reply:
Correct, I'll update it.